docker/recorder/src/main/java/org/arquillian/cube/docker/impl/client/utils/NumberConversion.java

```diff
@@ -6,13 +6,15 @@
 public class NumberConversion {
 
     public static String humanReadableByteCount(Long bytes, boolean decimal) {
-        int unit = decimal ? 1000 : 1024;
         if (bytes == null) bytes = 0L;
-        if (bytes < unit) return bytes + " B";
-        int exp = (int) (Math.log(bytes) / Math.log(unit));
+        String sign = bytes < 0 ? "-" : "";
+        Long absBytes = Math.abs(bytes);
+        int unit = decimal ? 1000 : 1024;
+        if (absBytes < unit) return sign + absBytes + " B";
+        int exp = (int) (Math.log(absBytes) / Math.log(unit));
         String pre = (decimal ? "kMGTPE" : "KMGTPE").charAt(exp-1) + (decimal ? "" : "i");
 
-        return String.format("%.2f %sB", bytes / Math.pow(unit, exp), pre);
+        return sign + String.format("%.2f %sB", absBytes / Math.pow(unit, exp), pre);
```

Comment:
You can add sign into `format` something like `String.format("%s %.2f......", sign, .....)`